crates/gitbutler-repo/src/repository.rs

```diff
@@ -449,10 +449,17 @@ impl RepoActionsExt for CommandContext {
 
 type OidFilter = dyn Fn(&git2::Commit) -> Result<bool>;
 
+/// Generally, all traversals will use no particular ordering, it's implementation defined in `git2`.
```

Comment:
I looked into replacing `l`  with `gitoxide`, but only ended up documenting this type as I though it's not really worth replacing something this trivial as it would need to on-demand open a `gix::Repository` again.

Once `merge_base` is also available, I think I might change the `CommandContext` to also open a `gix::Repository` lazily.